processing/data_collection/gazette/spiders/go_aparecida_de_goiania.py

```diff
@@ -0,0 +1,37 @@
+import datetime as dt
+from json import loads as json_load
```

Comment:
This is not a good practice for me. What the problem with using `json.loads()`?
When I see `json_load` in other part of the code, it is not explicit that this is the function from the standard library. My guess would be that you created a custom function to deserialize a JSON string.

`import json` and then use `json.loads()` when you need it.